README.md

```diff
@@ -1,2 +1,155 @@
-This file is to initialize the repo.
 # flutter_flux
+
```

Comment:
Please add a disclaimer that this is an experimental package and does not have official support from the Flutter team. But, feedback most welcome!